internal/store/helper.go

```diff
@@ -0,0 +1,117 @@
+package store
+
+import (
+	"context"
+	"fmt"
+	_ "github.com/golang-migrate/migrate/v4/database/pgx"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/kong/pg-aurora-client/pkg/pool"
+	"github.com/testcontainers/testcontainers-go"
```

Comment:
Neat library... I actually thought about building something like this & had no idea someone already did it. Bookmarking this...